lib/facter/facts/aix/serialnumber.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Facts
+  module Aix
+    class Serialnumber
+      FACT_NAME = 'serialnumber'
+
```

Comment:
this is a legacy fact and it should have TYPE = :legacy.  See linux/sshfp_algorithm